### DISCUSSION TITLE
page.waitForTimeout is not a function at callChrome when using `setDelay()` with Puppeteer >=22

### DISCUSSION THREAD

**Discussion by alancwoo** [spatie/browsershot#831] [Bugs]
If using `setDelay()` you get an error: `page.waitForTimeout is not a function at callChrome` with puppeteer `>= 22` as it was removed: https://github.com/puppeteer/puppeteer/pull/11780

They suggest alternatives, which would mean adjusting the library accordingly, but in the meantime I have resolved by forcing Puppeteer v21.x which is less than ideal.

**Comment by bessone** [comment]
Related to #681, up to now the officially supported version of Puppeteer is lower than 19, unless there are installation workarounds.

**Comment by arharp** [comment]
For anyone else who comes across this, #834 addressed this issue.
